pkg/infra/usagestats/statscollector/service.go

```diff
@@ -189,6 +196,15 @@ func (s *Service) collect(ctx context.Context) (map[string]interface{}, error) {
 		return nil, err
 	}
 
+	variants, err := s.detectPrometheusVariants(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	for variant, count := range variants {
+		m["stats.ds.prometheus.flavor."+variant+".count"] = count
```

Comment:
My first thinking was to write mimir here https://github.com/grafana/grafana/pull/47942/files#diff-591ea46a01054a12936a1fcaa885fd575e6414ac992ff8d09ed1b33cfe30fe79R163 but this is better :+1: